docs/src/documentation/03-components/12-layout/meta.yml

```
@@ -1,2 +1,3 @@
 title: Layout
+description: Components to help you lay out your content.
```

Maybe this would be slightly more informative.
```suggestion
description: Components to help you lay out your content on the page.
```

---

I've tried to stay away from the term "page" because it's less universal, but given the components here, it's probably fine.